packages/strapi-plugin-content-type-builder/admin/src/containers/Form/index.js

```diff
@@ -527,7 +527,15 @@ export class Form extends React.Component { // eslint-disable-line react/prefer-
   }
 
   renderModalBodyChooseAttributes = () => {
-    const attributesDisplay = has(this.context.plugins.toJS(), 'upload')
+    let plugins;
+
+    try {
+      plugins = this.context.plugins.toJS();
+    } catch(err) {
+      plugins = this.context.plugins;
+    }
+
+    const attributesDisplay = has(plugins, 'upload')
```

Comment:
```js
const attributesDisplay = plugins.upload !== undefined ?
  ... :
  ...;
```

Reply:
Same Here I have just fixed the bug nothing more. The content-type-builder needs a way better update than this line of code.

Reply:
I agree, we will do it later then